src/main/java/io/cdap/e2e/pages/locators/CdfSysAdminLocators.java

```diff
@@ -51,4 +53,92 @@ public class CdfSysAdminLocators {
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy='Clear All']")
   public static WebElement clearAll;
+  @FindBy(how = How.XPATH, using = "//span//button[contains(@class, 'btn btn-secondary')]")
+  public static WebElement reloadSystemArtifacts;
+  @FindBy(how = How.XPATH, using = "//a[@href='/cdap/httpexecutor']")
+  public static WebElement makeHttpCall;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='create-namespace-btn']")
+  public static WebElement createNewNamespace;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='edit-system-prefs-btn']")
+  public static WebElement editSystemPreference;
+  @FindBy(how = How.XPATH, using = "//a[@href='/cdap/ns/system/profiles/create']")
+  public static WebElement createComputeProfile;
+  @FindBy(how = How.XPATH, using = "//div//label[@for='import-profile']")
+  public static WebElement clickImport;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='save-prefs-btn']")
+  public static WebElement clickSaveClose;
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Namespace name' and @type='text']")
+  public static WebElement namespaceName;
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Namespace description' and @type='text']")
+  public static WebElement namespaceDescription;
+  @FindBy(how = How.XPATH, using = "//div[contains(@class,'namespaces')]")
+  public static WebElement openNamespace;
+  @FindBy(how = How.XPATH, using = "//div[contains(@class,'system-profiles')]")
+  public static WebElement openSystemComputeProfile;
+  @FindBy(how = How.XPATH, using = "//span[contains(@class,'reset')]")
+  public static WebElement clickReset;
+  @FindBy(how = How.XPATH, using = "//span[contains(@class,'success reset-success')]")
+  public static WebElement resetSuccessMsg;
+  @FindBy(how = How.XPATH, using = "//span//button[@type='submit' and @class='btn remove-row-btn btn-link']")
+  public static WebElement clickDelete;
+  @FindBy(how = How.XPATH, using = "//button[@data-testid='Reload']")
+  public static WebElement clickReload;
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'Close')]")
+  public static WebElement closeButton;
+  @FindBy(how = How.XPATH, using = "//div[contains(@class, 'error-section text-danger')]")
+  public static WebElement errorMessageOnFooter;
+  @FindBy(how = How.XPATH, using = "//div[contains(@class, 'card-action-feedback DANGER')]")
+  public static WebElement failMessage;
+
```

Comment:
A common comment for above locators, if we can find an Id for these locators we should use them. Avoid using such locator as much as possible. 

```@type='submit' and @class='btn remove-row-btn btn-link']```

Note - Class names are mostly defined by developers to apply styling/css on web pages and they can change them quite frequently while an UI update.